src/goto-instrument/code_contracts.h

```diff
@@ -98,12 +98,14 @@ class code_contractst
     const irep_idt &function_id,
     const irep_idt &mode);
 
-  void check_apply_invariants(
+  void check_apply_invariants_and_decreases(
```

Comment:
I'd just call it `check_apply_loop_contracts` now. Assigns clause stuff would also be added in this function eventually